src/main/java/org/killbill/billing/plugin/vertex/VertexApiClient.java

```diff
@@ -34,10 +34,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.killbill.billing.plugin.vertex.VertexConfigProperties.VERTEX_ADJUSTMENTS_LENIENT_MODE_PROPERTY;
+import static org.killbill.billing.plugin.vertex.VertexConfigProperties.VERTEX_COMPANY_DIVISION_PROPERTY;
```

Comment:
While introducing new property I revised the naming convention for properties:
- for those which are specific to Vertex O-Series API like credentials there is `_OSERIES_` in the name
- no `_OSERIES_` for data/logic properties

Reply:
i think we could have left company_name and division as is because its actually linked to how vertex is configured and is external to our system and managed by Vertex really.